include/odp/api/spec/packet.h

```diff
@@ -937,7 +937,7 @@ void *odp_packet_buf_head(odp_packet_buf_t pkt_buf);
 /**
```

Comment:
In this file, there seems to be a couple cases of single colon vs. double colon, maybe those could be fixed as well, `odp_packet_vector_tbl()` spec doc, second `@note` section (though, doxygen seems to have troubles creating links to unnamed struct/union members without extra tricks).

Reply:
Sure, fixed in V2.